tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -46,10 +46,24 @@ const assertContentOFLocalFileIs = function (fullPathOflocalFile, expectedConten
   )
 }
 
-Before(function (client) {
+Before(function () {
   createdFiles = []
+  if (typeof process.env.SCREEN_RESOLUTION !== 'undefined' && process.env.SCREEN_RESOLUTION.trim() !== '') {
+    let resolution = process.env.SCREEN_RESOLUTION.split('x')
+    resolution[0] = parseInt(resolution[0])
+    resolution[1] = parseInt(resolution[1])
+    if (resolution[0] > 1 && resolution[1] > 1) {
+      client.resizeWindow(parseInt(resolution[0]), parseInt(resolution[1]))
```

Comment:
```suggestion
      client.resizeWindow(resolution[0], resolution[1])
```

`parseInt()` is redundant here

Reply:
just to make sure there is no random rubish in the string. its a user input - never trust the user

Reply:
you are right, there is a parse before that also, will change it

Reply:
@paurakhsharma done